### PR TITLE
Update ADL Java SDK to latest Swagger

### PR DESCRIPTION
**Pull request by begoldsm**
This update does the following:
* Update to latest swagger which includes:
  - Firewall Rule enable/disable
  - Trusted Identity Provider Support
  - Flattening of nested properties
  - Inclusion of commitment tier for accounts
  - Encryption support for ADLS accounts
* Update to the Beta5 version of runtime
* Update all tests to use the recording model and create recordings.
**NOTE: There is a bug in the generator, which resulted in a manual fix to
the generated code: https://github.com/Azure/autorest/issues/1760
That issue must be fixed or the manual fix up of ADL sub resources must
be done on the next regeneration.**

### CODE REVIEW

File: azure-mgmt-datalake-store/src/test/java/com/microsoft/azure/management/datalake/store/DataLakeStoreManagementTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.microsoft.azure.management.datalake.store;
```
Comment (hovsepm) [inline azure-mgmt-datalake-store/src/test/java/com/microsoft/azure/management/datalake/store/DataLakeStoreManagementTest.java]: >package com [](start = 0, length = 11)

Please add license header....

**Comment by hovsepm** [comment]
> package com.microsoft.azure.management.datalake.store;

Here as well. License header is missing. 

---
Refers to: azure-mgmt-datalake-store/src/test/java/com/microsoft/azure/management/datalake/store/DataLakeStoreAccountOperationsTests.java:1 in 4cb3989. [](commit_id = 4cb39893091888c7241cef15fc3ac946aa7d7143, deletion_comment = False)

File: azure-mgmt-datalake-analytics/src/test/java/com/microsoft/azure/management/datalake/analytics/DataLakeAnalyticsManagementTestBase.java [hunk 2]
```diff
@@ -1,117 +1,147 @@
 package com.microsoft.azure.management.datalake.analytics;
```
Comment (hovsepm) [inline azure-mgmt-datalake-analytics/src/test/java/com/microsoft/azure/management/datalake/analytics/DataLakeAnalyticsManagementTestBase.java]: >microsoft [](start = 12, length = 9)

License header missing.

**Comment by hovsepm** [comment]
> package com.microsoft.azure.management.datalake.analytics;

License header missing. 

---
Refers to: azure-mgmt-datalake-analytics/src/test/java/com/microsoft/azure/management/datalake/analytics/DataLakeAnalyticsJobOperationsTests.java:1 in 4cb3989. [](commit_id = 4cb39893091888c7241cef15fc3ac946aa7d7143, deletion_comment = False)

**Comment by hovsepm** [comment]
> 

License header missing. 

---
Refers to: azure-mgmt-datalake-analytics/src/test/java/com/microsoft/azure/management/datalake/analytics/DataLakeAnalyticsCatalogOperationsTests.java:2 in 4cb3989. [](commit_id = 4cb39893091888c7241cef15fc3ac946aa7d7143, deletion_comment = False)

**Comment by hovsepm** [comment]
> 

License header missing. 

---
Refers to: azure-mgmt-datalake-analytics/src/test/java/com/microsoft/azure/management/datalake/analytics/DataLakeAnalyticsAccountOperationsTests.java:2 in 4cb3989. [](commit_id = 4cb39893091888c7241cef15fc3ac946aa7d7143, deletion_comment = False)

**Comment by begoldsm** [comment]
@hovsepm license header has been added to all test files. Can you take a look and get this merged?

**Pull request merged**